python/google/protobuf/json_format.py

```diff
@@ -88,7 +88,8 @@ class ParseError(Error):
 
 def MessageToJson(message,
                   including_default_value_fields=False,
-                  preserving_proto_field_name=False):
+                  preserving_proto_field_name=False,
+                  omitting_insignificant_whitespace=False):
```

Comment:
I wouldn't mind to change if all think it's better. 

On one hand It's good because it would serve "two purposes", indent or not and how many spaces to use for indentation, but maybe a it's a bit less explict (is it weird to right `MessageToJson(some_dict, indent=None)`? I don't think it is actually), and the point raised by @linyatis is also valid (same behavior between different implementations).